dnsimple/registrar.go

```diff
@@ -122,6 +121,9 @@ type DomainRegisterRequest struct {
 	// Set to true to enable the auto-renewal of the domain.
 	// Default to true.
 	EnableAutoRenewal bool `json:"auto_renew,omitempty"`
+	// Required as confirmation of the price, only if the domain is premium.
+	// Default to true.
```

Comment:
I don't understand why this says "Default to true" when this is a string value.

Reply:
It is a mistake.